docs/extensions/adi_links.py

```diff
@@ -11,7 +11,7 @@
 dft_url_dokuwiki	= 'https://wiki.analog.com'
 dft_url_ez			= 'https://ez.analog.com'
 dft_url_git			= 'https://github.com/analogdevicesinc'
-dft_url_part		= 'https://www.analog.com/products'
+dft_url_part		= 'https://www.analog.com'
```

Comment:
This was shortsighted on my part, thinking of dropping this "part" role and rename it to "adi", e.g.

```
:adi:`ad7172-2`
```
but I can patch after your PR